core/tests/protractor_utils/editor.js

```diff
@@ -402,7 +402,7 @@ var _setRuleParameters = function(ruleElement, interactionId, ruleName) {
   for (var i = 0; i < parameterValues.length; i++) {
     var parameterElement = ruleElement.all(
       by.css('.protractor-test-answer-description-fragment'
-    )).get(i * 2 + 1);
+      )).get(i * 2 + 1);
```

Comment:
Again, in multiline brackets, break after '('

Might be better to move by.css( to the previous line

Reply:
Got it.